recipes/nni/meta.yaml

```
@@ -0,0 +1,38 @@
+{% set name = "nni" %}
+{% set version = "2.10" %}
+{% set sha256 = "826702c4165731c044d30af49230a761ffc753bda8f6be38be9be642fd367037" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/microsoft/nni/archive/refs/tags/v2.10.tar.gz
```

it's better to use the variable defined on line 2:
```suggestion
  url: https://github.com/microsoft/nni/archive/refs/tags/v{{ version }}.tar.gz
```